src/operator/controllers/linkerd/linkerd_manager.go

```diff
@@ -0,0 +1,757 @@
+				err = ldm.Client.Create(ctx, server)
+				if err != nil {
+					logrus.Errorf("Failed to create Linkerd server: %s", err.Error())
+					return nil, errors.Wrap(err)
+				}
+			}
+			currentResources.Servers.Add(server.UID)
+
+			httpResources := target.GetHTTPResources()
+			if len(httpResources) > 0 {
+				probePath, err := ldm.getLivenessProbePath(pod) // should be get livenessprobepath for container with port
+				if err != nil {
+					return nil, errors.Wrap(err)
+				}
+
+				if probePath != "" {
+					if err := ldm.handleLivenessProbResources(ctx, probePath, clientNamespace, server.Name, svcIdentity, target, port, currentResources); err != nil {
+						return nil, errors.Wrap(err)
+					}
+				}
+
+				for _, httpResource := range httpResources {
+					if err := ldm.handleHTTPResource(ctx, clientNamespace, server.Name, target, svcIdentity, httpResource, port, currentResources); err != nil {
+					}
+				}
+			} else {
+				policy, shouldCreatePolicy, err := ldm.shouldCreateAuthPolicy(ctx, ep.Service,
+					server.Name,
+					LinkerdServerKindName,
+					"meshtls-for-client-"+ep.Service.GetName(),
+					LinkerdMeshTLSAuthenticationKindName)
+				if err != nil {
+					return nil, errors.Wrap(err)
+				}
+
+				if shouldCreatePolicy {
+					policy = ldm.generateAuthorizationPolicy(ep.Service, target.Target,
+						port,
+						server.Name,
+						LinkerdServerKindName,
+						LinkerdMeshTLSAuthenticationKindName)
+					err = ldm.Client.Create(ctx, policy)
+					if err != nil {
+						logrus.Errorf("Failed to create Linkerd policy: %s", err.Error())
+						return nil, errors.Wrap(err)
+					}
+				}
+				currentResources.AuthorizationPolicies.Add(policy.UID)
+			}
+		}
+	}
+	return currentResources, nil
+}
+
+func (ldm *LinkerdManager) BuildPodLabelSelectorFromTarget(target otterizev2alpha1.Target, intentsObjNamespace string) metav1.LabelSelector {
+	otterizeServerLabel := map[string]string{}
+	targetServerIdentity := target.ToServiceIdentity(intentsObjNamespace)
+
+	formattedTargetServer := targetServerIdentity.GetFormattedOtterizeIdentityWithoutKind()
+	otterizeServerLabel[otterizev2alpha1.OtterizeServiceLabelKey] = formattedTargetServer
+
+	return metav1.LabelSelector{MatchLabels: otterizeServerLabel}
+}
+
+func (ldm *LinkerdManager) getServerName(intent otterizev2alpha1.Target, port int32) string {
+	name := intent.GetTargetServerName()
+	return fmt.Sprintf(OtterizeLinkerdServerNameTemplate, name, port)
+}
+
+func (ldm *LinkerdManager) createIntentPrimaryResources(ctx context.Context, svcIdentity serviceidentity.ServiceIdentity, clientServiceAccount string) error {
+	//shouldCreateNetAuth, err := ldm.shouldCreateNetAuth(ctx, intents)
+	//if err != nil {
+	//	return err
+	//}
+	//
+	//if shouldCreateNetAuth {
+	//	netAuth, err := ldm.generateNetworkAuthentication(ctx, intents)
+	//	if err != nil {
+	//		return err
+	//	}
+	//	err = ldm.Client.Create(ctx, netAuth)
+	//	if err != nil {
+	//		return err
+	//	}
+	//}
```

Comment:
@aerosouund is this still needed ? I didn't see any mention of this in docs/tutorials. I wasn't sure if this resource was relevant still.